src/main/scala/org/dia/algorithms/mcc/GTGRunner.scala

```diff
@@ -348,11 +350,20 @@ class GTGRunner(val masterURL: String,
     val subgraphs = sc.sparkContext.parallelize(edgeListRDDIndexed.toSeq)
       .map(MCCOps.mapEdgesToBuckets(_, maxParitionSize, buckets))
       .groupByKey()
-    val subgraphsFound = MCCOps.findSubgraphsIteratively(subgraphs, 1, maxParitionSize, minGraphLength, sc.sparkContext)
+    val subgraphsFound = MCCOps.findSubgraphsIteratively(subgraphs, 1, maxParitionSize,
```

Comment:
@sujen1412 @kwhitehall 
1. On Line 346, you don't need another edgeListRDD.collect().
    Look at the MCCEdgeList initialized on line 336.

2. We can avoid the parallelize call here.
MCCOps.createPartitionIndex can take an RDD instead of an iterable.
This requires some re-working of the createPartitionIndex method.
Look here for a possible example:
https://gist.github.com/rahulpalamuttam/9799b0152a9998db4fc955a078654b77